yarn.lock

```diff
@@ -1622,7 +1622,6 @@ acorn-globals@^3.1.0:
 acorn-jsx@^3.0.0:
   version "3.0.1"
   resolved "https://registry.yarnpkg.com/acorn-jsx/-/acorn-jsx-3.0.1.tgz#afdf9488fb1ecefc8348f6fb22f464e32a58b36b"
-  integrity sha1-r9+UiPsezvyDSPb7IvRk4ypYs2s=
```

Comment:
Are you using an old version of yarn? Weird that it's removing this line. Perhaps try removing the fine and regenerating?

Reply:
Just regenerated the lock file.
I added the volta config in all of the package.json files as well, so that should keep the proper yarn version.